bitnami/dataplatform-bp1/Chart.yaml

```
@@ -19,20 +19,25 @@ dependencies:
     name: zookeeper
     repository: https://charts.bitnami.com/bitnami
     version: 6.x.x
+  - condition: wavefront.enabled
+    name: wavefront
+    repository: https://charts.bitnami.com/bitnami
   - name: common
     repository: https://charts.bitnami.com/bitnami
     tags:
       - bitnami-common
     version: 1.x.x
 description: OCTO Data platform Kafka-Spark-Solr Helm Chart
 engine: gotpl
-home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform
+home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform-bp1
 keywords:
```

It is also needed to add the icon to be rendered in the different hubs and/or marketplaces:
```suggestion
home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform-bp1
icon: https://bitnami.com/assets/stacks/dataplatform-bp1/img/dataplatform-bp1-stack-220x234.png
keywords:
```